frontend/app/routes/$lang+/_public+/status+/index.tsx

```diff
@@ -50,6 +59,8 @@ export async function action({ context: { session }, params, request }: ActionFu
     errors?: z.ZodFormattedError<{ sin: string; code: string }, string>;
     formData?: Partial<z.infer<typeof formDataSchema>>;
     status?: string;
+    clientFriendlyStatus?: { id: string; nameEn: string; nameFr: string };
+    alertType?: 'warning' | 'success' | 'danger' | 'info';
   } = {};
```

Comment:
Remove this `response` object and return `json({errors})` and `json({status})` in both places. You will be able to do typescript narrowing to use either `errors` or `status` from the action data.
https://www.typescriptlang.org/docs/handbook/2/narrowing.html#the-in-operator-narrowing